layouts/header.html

```diff
@@ -40,6 +40,7 @@
 
     <!-- Custom styles for this template -->
     <link href="/css/docs.css" rel="stylesheet">
+    <link href="/css/routing-tree-editor.css" rel="stylesheet">
```

Comment:
We could of course add an `<%= if ... %>` check here if we really wanted. Like we do in https://github.com/prometheus/docs/blob/routing-tree-deux/layouts/header.html#L56.

But currently in docs.css we also have some styles that are only applicable for certain pages, so we're loading too much CSS already. It's not nice and we should optimize that, but I think it's small enough to still be ok for now.